src/main/java/VASSAL/build/GameModule.java

```diff
@@ -952,6 +953,12 @@ protected void checkGpIds() {
     for (PieceSlot pieceSlot : theModule.getAllDescendantComponentsOf(PieceSlot.class)) {
       checker.add(pieceSlot);
     }
+    // Add any PieceSlots in Prototype Definitions
+    for (PrototypesContainer pc : theModule.getComponentsOf(PrototypesContainer.class)) {
+      for (PrototypeDefinition pd : pc.getDefinitions()) {
+        checker.add(pd);
+      }
```

Comment:
I think you could do `pc.getDefinitions.forEach(checker::add);` here for the inner loop?